app/assets/stylesheets/alchemy/elements.scss

```diff
@@ -723,7 +723,7 @@ div.pictures_for_element {
   margin-top: 4px;
 }
 
-textarea.default_tinymce, textarea.custom_tinymce {
+textarea.has_tinymce {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector `has_tinymce` should be written in lowercase with hyphens